src/translations/screens/Nearby.ts

```diff
@@ -66,6 +66,10 @@ const NearbyTexts = {
     ),
     confirm: _('Bekreft', 'Confirm'),
     header: _('Velg dato og tid', 'Select date and time'),
+    a11yInPastHint: _(
+      'Valgt tidspunkt er i fortiden. Nåtid vil bli brukt i stedet.',
```

Comment:
Siden dette er på en button så burde hintet være mer formulert mot hva som skjer når man trykker på knappen, gjerne med "Aktivér for ..." som andre steder.

Når det er sagt, har vi tilsvarende visuell advarsel på dette for de som ikke bruker skjermleser? Hvis ikke så tenker jeg det ikke er nødvendig å opplyse om ved skjermleser heller.

Reply:
Ser den. Da heller jeg mot å fjerne det i denne omgang, og så ser vi om vi får lagt inn en advarsel uten skjermleser også.